drivers/core_demographics_report/app/models/core_demographics_report/details_column.rb

```diff
@@ -5,7 +5,7 @@ def value(row)
       raw_value = row[index]
 
       project_id = row[project_id_index]
-      policy = user.policies.for_project(project_id)
+      policy = user.policy_for(project_id, policy_class: GrdaWarehouse::AuthPolicies::ProjectPiiPolicy)
```

Comment:
Is the reason we can't use the client policy here is because we don't have client objects correct?  I wonder if we should pivot, load all the related clients and index them by id so we can use the appropriate policy.  Potentially that would require storing tens of thousands of client objects in memory, so maybe we can't. 

Looking back through, I think using the client policy may also pull down all of the project ids related to the client, which would be additional queries.    I also think this isn't wrong, it just ignores ROI correct?  So if a client is included here but you can't view their client name based on the project enrollment, but you _could_ if we checked ROI, it will be hidden.  If that's the only side effect, I think this is probably fine as is.